### PR TITLE
Update to use nats.requestOne

### PR DESCRIPTION
**Pull request by aricart**
- Bumped node nats library requirements.
- Updated Travis build to use the latest version of nats-streaming-server

### CODE REVIEW

File: lib/stan.js [hunk 1]
```diff
@@ -512,7 +513,15 @@ Stan.prototype.subscribe = function(subject, qGroup, options) {
...
-  var rsid = this.nc.request(this.subRequests, new Buffer(sr.serializeBinary()), {'max': 1}, function(msg) {
+  // fixme: hardcoded timeout
+  this.nc.requestOne(this.subRequests, new Buffer(sr.serializeBinary()), 2*1000, function(msg) {
+    if(msg instanceof nats.NatsError) {
+      if(msg.code === nats.REQ_TIMEOUT) {
+        that.emit('timeout', msg);
+      } else {
+        that.emit('error', msg);
+      }
+    }
```
Comment (kozlovic) [inline lib/stan.js]: Don't we need a return here?

File: lib/stan.js [hunk 2]
```diff
@@ -653,21 +657,28 @@ function closeOrUnsubscribe(that, doClose) {
...
+      var err;
+      if(msg instanceof nats.NatsError) {
+          var type = doClose ? CLOSE_REQ_TIMEOUT : UNSUB_REQ_TIMEOUT;
+          err = new nats.NatsError(type, type, msg);
+          if(msg.code === nats.REQ_TIMEOUT) {
+            that.emit('timeout', err);
+          } else {
+              that.emit('error', err);
+          }
+      }
```
Comment (kozlovic) [inline lib/stan.js]: Same than other. No return here?

File: .travis.yml [hunk 3]
```diff
@@ -5,18 +5,26 @@ node_js:
   - "4"
   - "6"
   - "8"
+  - "9"
+
+git:
+  depth: false
+
+env:
+  - STAN_VERSION=v0.8.0-beta
```
Comment (kozlovic) [inline .travis.yml]: You could also use v0.7.2 to use the latest stable version.

Comment (aricart) [reply .travis.yml]: Excellent catch on the return - tests seem to run correctly with the beta. More important is the triggering of node-nats-streaming on node-nats changes.

**Pull request merged**